cmake/setup_withdeps.sh.in

```diff
@@ -20,7 +20,11 @@ else
 fi
 
 # source the python and ODD environment
-. $script_dir/python/setup.sh
+if [[ -f "$script_dir/python/setup.sh" ]]; then
+  . "$script_dir/python/setup.sh"
+else
+  echo "WARNING: setup.sh for python and ODD is missing."
+fi
```

Comment:
I think we could do one better by checking if Python bindings are enabled and then sourcing the script. Right now it would always source the script if it exists even if it might be outdated and the warning might not be justified as the user could have disabled the bindings.